<launch>
    <group ns="turtlesim1">
        <node pkg="turtlesim" name="sim" type="turtlesim_node"/>
        <node pkg="turtlesim" name="tele" type="turtle_teleop_key"/>
        <node pkg="robots" name="zombie" type="zombie_turtle.py">
            <param name="speed_linear" value="1.0"/>
            <param name="speed_angular" value="1.0"/>
        </node>
    </group>

    <!-- <node pkg="turtlesim" name="mimic" type="mimic">
        <remap from="input" to="turtlesim1/turtle1"/>
        <remap from="output" to="turtlesim2/turtle1"/>
    </node> -->
</launch>